Lib/test/test_netrc.py

```diff
@@ -123,6 +123,20 @@ def test_security(self):
             os.chmod(fn, 0o622)
             self.assertRaises(netrc.NetrcParseError, netrc.netrc)
 
+    def test_file_not_found_in_home(self):
+        d = support.TESTFN
+        os.mkdir(d)
+        self.addCleanup(support.rmtree, d)
+        with support.EnvironmentVarGuard() as environ:
+            environ.set('HOME', d)
+            self.assertRaises(FileNotFoundError, netrc.netrc)
+
+    def test_file_not_found_explicit(self):
+        unlikely_file = "unlikely_netrc"
```

Comment:
No need to use `unlikely_file` here. Just pass `"unlikely_netrc"` to the constructor.